ios/gutenberg/GutenbergViewController.swift

```diff
@@ -194,6 +196,33 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
     func gutenbergDidLogUserEvent(_ event: GutenbergUserEvent) {
         print("Gutenberg loged user event")
     }
+
+    func gutenbergDidRequestUnsupportedBlockFallback(for block: Block) {
+        print("Requesting Fallback for \(block)")
+        let controller = try! WebViewController(block: block, userId: "0")
```

Comment:
Currently yes, since `WebViewController` is pretty much empty.
But I'm thinking that we might use it if we make automated tests. We would need a way to change the target Editor URL.

Reply:
Gotcha 👍